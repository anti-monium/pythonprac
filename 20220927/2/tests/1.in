15, 17, 1, 99, 27, 88, 11, 33, 56, 31, 23, 46, 76, 64, 25, 34, 7, 4, 12, 9, 100, 2, 64
