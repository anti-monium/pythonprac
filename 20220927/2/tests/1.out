[100, 99, 1, 2, 46, 4, 11, 25, 15, 23, 27, 56, 88, 12, 7, 34, 31, 76, 9, 33, 17, 64, 64]
